### PR TITLE
[INPUT] Update Czech (cs-CZ) translation

### PR DESCRIPTION
**Pull request by Venca24**
## Purpose

Updates the Czech translation of language control panel.

JIRA issue: none

### CODE REVIEW

File: dll/cpl/input/lang/cs-CZ.rc [hunk 1]
```diff
@@ -1,6 +1,9 @@
-/* FILE:       dll/cpl/input/lang/cs-CZ.rc
- * TRANSLATOR: Radek Liska aka Black_Fox (radekliska at gmail dot com)
- * UPDATED:    2015-04-12
+/*
+ * PROJECT:     ReactOS Language Control Panel
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Czech resource file
+ * TRANSLATORS: Copyright 2015 Radek Liska (Black_Fox) <radekliska@gmail.com>
```
Comment (binarymaster) [inline dll/cpl/input/lang/cs-CZ.rc:5]:
```suggestion
 * TRANSLATORS: Copyright 2010-2015 Radek Liška (Black_Fox) <radekliska@gmail.com>
```

File: dll/cpl/input/lang/cs-CZ.rc [hunk 2]
```diff
@@ -10,10 +13,10 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "Nastavení"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    GROUPBOX "Default input &language", -1, 5, 5, 246, 55
-    LTEXT "Select one of the installed input languages to use when your computer starts.", -1, 15, 15, 230, 23
+    GROUPBOX "Výchozí vstupní &jazyk", -1, 5, 5, 246, 55
+    LTEXT "Vyberte jeden z nainstalovaných jazyků, který se bude používat po spuštění Vašeho počítače.", -1, 15, 15, 230, 23
```
Comment (HBelusca) [inline dll/cpl/input/lang/cs-CZ.rc:17]: The (longer) string still fits OK in the window?

**Review by HBelusca — APPROVED** [review]

File: dll/cpl/input/lang/cs-CZ.rc (hunk 2 above)
Comment (Venca24) [reply dll/cpl/input/lang/cs-CZ.rc:17]: @HBelusca I would say, yes.
Screenshot_live_test_2024-07-06_22:50:53

Comment (JoachimHenze) [reply dll/cpl/input/lang/cs-CZ.rc:17]: Resolved.

**Review by JoachimHenze — APPROVED** [review]
Approved.

File: dll/cpl/input/lang/cs-CZ.rc [hunk 3]
```diff
@@ -10,10 +13,10 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
...
     COMBOBOX IDC_DEFAULT_LANGUAGE, 15, 40, 230, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-    GROUPBOX "&Installed services", -1, 5, 65, 246, 125
+    GROUPBOX "Na&instalované služby", -1, 5, 65, 246, 125
```
Comment (JoachimHenze) [inline dll/cpl/input/lang/cs-CZ.rc:19]: Picture please that proved the **MS** cs-CZ does use a small &i here. They do typically recommend to avoid that.

Comment (Venca24) [reply dll/cpl/input/lang/cs-CZ.rc:19]: @JoachimHenze It's actually on the "s".
Screenshot of Windows XP' language control panel in Czech

Comment (JoachimHenze) [inline dll/cpl/input/lang/cs-CZ.rc:19]: Well, then let's place it at the &s for ros as well. I checked: the &s isn't used anywhere else in the dlg.
```suggestion
    GROUPBOX "Nainstalované &služby", -1, 5, 65, 246, 125
```

Comment (Venca24) [reply dll/cpl/input/lang/cs-CZ.rc]: I've also changed other strings according to the screenshot.

Comment (JoachimHenze) [reply dll/cpl/input/lang/cs-CZ.rc]: I checked them regarding accels. LGTM. Resolved, and final approval hereby.

**Pull request merged**